qiita_ware/search.py

```diff
@@ -0,0 +1,59 @@
+from collections import Counter
+
+from future.utils import viewvalues
+
+from qiita_db.search import QiitaStudySearch
+
+
+def count_metadata(results, meta_cols):
```

Comment:
this is not counting anything, so the name and the description are misleading. Can you change the name to something like `collapse_search_results` or something similar? Also I'm unsure if this function should be exposed to qiita_pet or should be private and be included in the `search` function below, which will make it useful.